grafana-plugin/e2e-tests/outgoingWebhooks/simpleWebhook.test.ts

```diff
@@ -0,0 +1,76 @@
+import express from 'express';
+import ngrok from 'ngrok';
+
+import { expect, test } from '../fixtures';
+import { clickButton, generateRandomValue } from '../utils/forms';
+import { goToOnCallPage } from '../utils/navigation';
+import { checkWebhookPresenceInTable } from '../utils/outgoingWebhooks';
+import { EscalationStep, createEscalationChain } from '../utils/escalationChain';
+import { createIntegrationAndSendDemoAlert } from '../utils/integrations';
+
+const createWebhook = async ({ page, webhookName, webhookUrl }) => {
+  await goToOnCallPage(page, 'outgoing_webhooks');
+
+  await clickButton({ page, buttonText: 'New Outgoing Webhook' });
+
+  await page.getByText('Simple').first().click();
+
+  await page.waitForTimeout(2000);
+
+  await page.keyboard.insertText(webhookUrl);
+  await page.locator('[name=name]').fill(webhookName);
+  await page.getByLabel('New Outgoing Webhook').getByRole('img').nth(1).click(); // Open team dropdown
+  await page.getByLabel('Select options menu').getByText('No team').click();
+  await clickButton({ page, buttonText: 'Create' });
+};
+
+test.describe('simple webhook', () => {
+  test('Create and check it is displayed on the list correctly', async ({ adminRolePage: { page } }) => {
+    const webhookName = generateRandomValue();
+    await createWebhook({ page, webhookName, webhookUrl: 'https://example.com' });
+    await checkWebhookPresenceInTable({ page, webhookName, expectedTriggerType: 'Escalation step' });
+  });
```

Comment:
this test was pre-existing in `grafana-plugin/e2e-tests/outgoingWebhooks/createSimpleWebhook.test.ts`, renamed the file + added the test below 👇